com.unity.visualeffectgraph/Documentation~/Reference-Attributes.md

```diff
@@ -23,15 +23,14 @@ Some attributes are a bit more advanced and will be used by default in most simu
 
 | Name       | Type    | Description                                                  | Default Value                                     |
 | ---------- | ------- | ------------------------------------------------------------ | ---------- |
-| `mass` | float | The mass of a particle in Kg/dm^3 | 1.0 (defaults to 1kg per liter of water) |
-| `angle` | Vector3 | **Variadic:** Euler rotation of a simulated element, expressed as a Vector of Degrees Values. | (0,0,0) |
+| `mass` | float | The mass of a particle | 1.0|
 | `angularVelocity` | Vector3 | **Variadic:** Euler rotation speed of a simulated element, expressed as a Vector of Degrees per second values. | (0,0,0) |
 | `oldPosition` | Vector3 | **Deprecated:** This attribute is a storage Helper if you want to back-up current position of a simulated element, before integrating its velocity. | (0,0,0) |
-| `targetPosition` | Vector3 | This attribute has various purposes: it can be a storage Helper if you want to store a position to reach, then compute a vector in order to reach this Target Position. In Line Renderers, this attribute can be also used to set the end of each line particle. | (0,0,0) |
+| `targetPosition` | Vector3 | This attribute has various purposes: it can be a storage Helper if you want to store a position to reach, then compute a vector to reach this Target Position. In Line Renderers, this attribute can be also used to set the end of each line particle. | (0,0,0) |
```

Comment:
I know this isn't a general update but this description really bothers me so I'm calling it out. I've improved it without getting too much into the technical aspects. I removed the bit about computing a vector to reach it because that process is a seperate use case example of what this attribute actually does, namely, storing the thing: 

"Use this attribute as a Helper to store a Target Position. You can also use it in Line Renderers to set the end of each line particle."